### PR TITLE
Added code to leverage loadDefaultPortal API

### PR DESCRIPTION
**Pull request by vishalshrm539**
* Added and modified the code to leverage loadDefaultPortal API.
* Made the changes to make SDK-A look similar to SDK-R.
* Will make similar changes in SDK-WC once this is approved as SDK-WC also has similar code.

### CODE REVIEW

File: src/app/_samples/full-portal/top-app-mashup/top-app-mashup.component.ts [hunk 1]
```diff
@@ -242,12 +243,25 @@ export class TopAppMashupComponent implements OnInit {
...
+  
+    // Note: myLoadPortal and myLoadDefaultPortal are set when bootstrapWithAuthHeader is invoked
+    if(thePortal){
+      console.log(`Loading specified appPortal: ${thePortal}`);
       window.myLoadPortal("app-root", thePortal, [], null);   // this is defined in bootstrap shell that's been loaded already
-    })
-
+    }else if(window.myLoadDefaultPortal){
+      console.log(`Loading default portal`);
+      window.myLoadDefaultPortal("app-root", [], null);
```
Comment (vrseraphin) [inline src/app/_samples/full-portal/top-app-mashup/top-app-mashup.component.ts:254]: I noticed we don't include that null final argument in SDK-R.  Might it be left off for SP-A as well.  I suspect undefined would be treated the same as null within JS Engine for that argument.  (same for myLoadPortal)

File: src/app/_services/auth.service.ts [hunk 2]
```diff
@@ -157,8 +157,9 @@ export class AuthService {
       //  So, create a reference to it
       window.myLoadMashup = bootstrapShell.loadMashup;
 
-      // For experimentation, save a reference to loadPortal, too!
+      // For experimentation, save a reference to loadPortal,loadDefaultPortal too!
```
Comment (vrseraphin) [inline src/app/_services/auth.service.ts:160]: We can eliminate the "For experimentation, " as no longer an experiment but we are actually using this approach

File: src/app/_services/server-config.service.ts [hunk 3]
```diff
@@ -121,13 +121,14 @@ export class ServerConfigService {
...
-          return;
-    }
+    
+    /** Commenting it out since we won't call selectPortal() if appPortal is specified in sdk-config.json */
+    // if ((serverConfig.appPortal !== "") &&
+    //     (serverConfig.appPortal !== undefined) ) {
+    //       // use the specified portal
+    //       console.log(`Using appPortal: ${serverConfig.appPortal}`);
+    //       return;
+    // }
```
Comment (vrseraphin) [inline src/app/_services/server-config.service.ts:132]: I believe this code should be left as is (not commented out).  Any harm in not commenting it out?  There is the entire path of when this new defaultLoadPortal API is not available (prior to 8.8.1 for now).

File: src/app/_samples/full-portal/top-app-mashup/top-app-mashup.component.ts (hunk 1 above)
Comment (vishalshrm539) [reply src/app/_samples/full-portal/top-app-mashup/top-app-mashup.component.ts:254]: Yes, null is treated as undefined as they're doing a truthy check. We don't pass in SDK-R...Ok, I'll remove.

File: src/app/_services/server-config.service.ts (hunk 3 above)
Comment (vishalshrm539) [reply src/app/_services/server-config.service.ts:132]: No harm but we won't call this method when "appPortal" is there in sdk-config.json. I think it was being used only in that case. But, I'll uncomment.

**Review by JEAxero — APPROVED** [review]
Ran tests against 8.7 and 8.8 and looks good

**Review by vrseraphin — APPROVED** [review]

**Pull request merged**